tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
OT: You mean `WDYT`, don't you? Because http://www.urbandictionary.com/define.php?term=wdyf 😀 

I'm rewriting everything with @julienfalque suggestions: multiline comments will be fixed like now, but with an option `false` by default.
The fixer by default will only fix PHPDocs, and so here a new question: by default it will be a PHPDoc related fixer, but an option will fix non-PHPDoc aka multiline comments, should be:

1. Keep it under `Comment` namespace
1. Move it to `Phpdoc` namespace
1. Split it

?